src/modules/crud.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-redeclare */
+/* eslint-disable block-scoped-var */
+/* eslint-disable no-var */
+/* eslint-disable vars-on-top */
+const list = document.getElementById('todos-list');
+const addInput = document.getElementById('todo-input');
+
+export const createTodo = () => {
+  const text = addInput.value;
+  const getLocalStorageData = localStorage.getItem('New Todo');
+  if (getLocalStorageData == null) {
+    var listArray = [];
+  } else {
+    var listArray = JSON.parse(getLocalStorageData);
+  }
+  listArray.push(text);
+  localStorage.setItem('New Todo', JSON.stringify(listArray));
+
+  if (text === '') {
+    return;
+  }
+
+  const li = document.createElement('li');
+
+  const checkbox = document.createElement('input');
+  checkbox.classList.add('checkbox');
+  checkbox.type = 'checkbox';
+
+  const paragraph = document.createElement('p');
+  paragraph.classList.add('paragraph');
+  paragraph.textContent = text;
+
+  const div = document.createElement('div');
+  div.classList.add('TextItems');
+
+  const remove = document.createElement('span');
```

Comment:
- [ ] When elements are removed they are still reflected in the local storage. Kindly ensure that removed items are deleted from the local storage.

**Deleting error:**
![image](https://user-images.githubusercontent.com/12745474/149507913-17b607c1-230f-46d1-b9ed-3aaac9ee91c7.png)